.github/dependabot.yml

```diff
@@ -4,3 +4,8 @@ updates:
     directory: /
     schedule:
       interval: daily
+
+  - package-ecosystem: npm
+    directory: /
+    schedule:
+      interval: daily
```

Comment:
I'm not sure about this one @MichaelDeBoey. In my opinion, examples should instead be thoughtfully updated, when it makes sense to do so:

- When we want to recommend a new Remix feature and we patch all examples (example: `loader` type inference)
- When we want to enhance the example itself, and leverage a new feature from the showcased integration/lib

Apps in this repo are not supposed to be using cutting edge dependencies. Users will just see how it works, before manually installing stuff into their own apps. 

I don't think we want to add more maintenance burden on our shoulders.. (especially with notifications from `dependabot`, daily, on 73 apps and growing 🙈)

wdyt ?

Reply:
Or is this just for the root package.json file that I've just seen now in your other merged PR? 😅

Reply:
If I understand the use of `directory` correctly, this should only give updates for the root `package.json` & that's what this PR is meant to do: automatically create PRs for dependencies of the root's `package.json`

I agree that examples should be updated manually.